### PR TITLE
Add summary output file to morph_check.

### PR DESCRIPTION
**Pull request by juanchopanza**
Output file name controlled with option -o or --output.
JSON format file with pairs of testname : PASS/FAIL for each
tested file. Client code can unpack this into less verbose
representations (e.g CSV tables.)

### CODE REVIEW

File: apps/morph_check [hunk 1]
```diff
@@ -132,24 +137,29 @@ def log_msg(ok, msg, color=False):
...
+    if summary_dict is not None:
+        summary_dict[msg] = 'PASS' if ok else 'FAIL'
+
 
 def test_file(f, config):
     '''Run tests on a morphology file'''
 
     L.info('Check file %s...', f)
 
+    summary_ = {}
```
Comment (mgeplf) [inline apps/morph_check]: why the trailing _?

Comment (juanchopanza) [inline apps/morph_check]: pylint. `summary` is used in `__main__`.

File: apps/morph_check [hunk 2]
```diff
@@ -231,6 +247,9 @@ if __name__ == '__main__':
 
     L.info(SEPARATOR)
 
+    with open(args.output_file, 'w') as json_output:
+        json.dump(summary, json_output)
```
Comment (mgeplf) [inline apps/morph_check]: I'd set `indent` so that it's more human readable.

**Comment by mgeplf** [comment]
Looks fine to me.

File: apps/morph_check (hunk 1 above)
Comment (mgeplf) [inline apps/morph_check]: Ah, I would make __main__ shorter and just call main().  Makes it easier to test, too

Comment (juanchopanza) [inline apps/morph_check]: I will do that in a future pull request. There's a lot here that needs cleaning up.

**Pull request merged**